.travis.yml

```diff
@@ -28,7 +28,7 @@ script:
   - |
     echo "Enable coverage report"
     TEST_ARG="-coverprofile=coverage.txt -covermode=atomic -coverpkg=all"
-  - go test ${TEST_ARG} ./...
+  - go test ${TEST_ARG} ./ ./presigner ./internal/...
```

Comment:
We can't include example packages in `go test` because it causes `flag redefined` panic. Probably that is a similar issue with this one https://github.com/mattn/goveralls/issues/89

Reply:
Adding build tag to the files to be excluded would be cleaner.
```diff
diff --git a/example/mqtt/main.go b/example/mqtt/main.go
index b1ecbfe..b52e638 100644
--- a/example/mqtt/main.go
+++ b/example/mqtt/main.go
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build examples
+
 /**
   Usage: go run . -thing-name NAME \
                   -private-key-path path/to/private-key \
diff --git a/example/wss/main.go b/example/wss/main.go
index 54465a7..be1ed45 100644
--- a/example/wss/main.go
+++ b/example/wss/main.go
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build examples
+
 /**
   Usage: go run . -host hoge-ats.iot.ap-northeast-1.amazonaws.com
 */
```
`go test ./...` doesn't build these files without adding `-tags=examples`

Reply:
Sorry, it's inconvenient to build example. (requires `go build . -tags=example`)

How about 
```
go test $(go list ./... | grep -v "/example/")
```
?
I think it would be better to test basically all files excepting some specific things.